index.php

```diff
@@ -0,0 +1,9 @@
+<?php
+require('/var/www/html/onboardingBelen/vendor/autoload.php');
```

Comment:
```suggestion
require('/var/www/html/onboardingBelen/vendor/autoload.php');
```
Base path root should be a config var, so when configured it should be accessible for everyone downloaded the project